src/adapters/autocomplete.js

```diff
@@ -99,76 +115,29 @@ SearchInput.prototype.select = async function(selectedPoi) {
   }
 }
 
+function remotesRender(pois) {
+  return pois.map(poi => renderItem(poi)).join('')
+}
+
+function favoritesRender(pois) {
+  return `<h3 class="autocomplete_suggestion__category_title">${_('FAVORIS', 'autocomplete')}</h3> ${pois.map(poi => renderItem(poi)).join('')}`
+}
+
 /* select sub template */
-function AutocompleteTemplate(poi) {
-  let content = ''
-  let {id, name, fromHistory, className, subClassName, type} = poi
+function renderItem(poi) {
+  let {id, name, fromHistory, className, subClassName, type, alternativeName} = poi
   let icon = IconManager.get({className : className, subClassName : subClassName , type : type})
   let iconDom = `<div style="color:${icon ? icon.color : ''}" class="autocomplete-icon ${`icon icon-${icon.iconClass}`}"></div>`
 
-  switch(poi.type) {
-    case 'poi':
-      content = PoiTemplate(poi, iconDom)
-      break
-    case 'house':
-      content = HouseTemplate(poi, iconDom)
-      break
-    case 'street':
-      content = StreetTemplate(poi, iconDom)
-      break
-    default: /* admin */
-      content = AdminTemplate(poi, iconDom)
-  }
-
    return `
 <div class="autocomplete_suggestion${fromHistory ? ' autocomplete_suggestion--history' : ''}" data-id="${id}" data-val="${ExtendedString.htmlEncode(name)}">
-  ${content}
-</div>
-`
-}
-
-function PoiTemplate(poi, iconDom) {
-  let {name, addressLabel} = poi
-  return `
-<div class="autocomplete_suggestion__first_line__container">
+  <div class="autocomplete_suggestion__first_line__container">
   ${iconDom}
   <div class="autocomplete_suggestion__first_line">${ExtendedString.htmlEncode(name)}</div>
 </div>
-${addressLabel ? `<div class="autocomplete_suggestion__second_line">${ExtendedString.htmlEncode(addressLabel)}</div>` : ''}`
-}
-
-function HouseTemplate(poi, iconDom) {
-  let {name, postcode, city, countryName} = poi
-  let address = [postcode, city, countryName].filter((zone) => zone).join(', ')
-  return `
-<div class="autocomplete_suggestion__first_line__container">
-  ${iconDom}
-  <div class="autocomplete_suggestion__first_line">${ExtendedString.htmlEncode(name)}</div>
+${alternativeName ? `<div class="autocomplete_suggestion__second_line">${ExtendedString.htmlEncode(alternativeName)}</div>` : ''}
```

Comment:
I think it would be cleaner to always display the "second_line" div, even if it's empty. It would give a consistent height to all autocomplete results.

Reply:
I'll give a hint (updated screenshot incoming)

Reply:
![acceptable](https://user-images.githubusercontent.com/5485051/50632581-a01fa080-0f48-11e9-9d7a-c270f1dc3d52.png)